.changeset/silver-cars-type.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-github-actions': minor
+---
+
+add support for GHES hosted repositories
```

Comment:
```suggestion
Added support GitHub Enterprise hosted repositories.

**BREAKING**: The `GithubActionsClient` is updated to take an `scmAuthApi` instead of the previous `githubAuthApi`. This does not require any code changes unless you construct your own `GithubActionsClient`
```